[DOP-2330] - add KeyValueHwm,  KeyValueIntHwm classes (#71)
MobileTeleSystems/etl-entities: docs/changelog/next_release/71.breaking.rst

> Remove ``cover`` method from base ``HWM`` class and its use with following tests in child classes

Reviewer: ```suggestion
Remove ``cover`` method from base ``HWM`` class, it only remains in ``FileHWM`` class
```
Надо добавить в `FileHWM` метод `covers` как абстрактный. На него завязана работа FileDownloader
